AuRoRA/src/scs/scs/sdv.py

```
@@ -0,0 +1,211 @@
+                archive_location: Path = Path(loaded_archive).parent             # Retrieve the location of the queued archive
+                
+                # SWEEP LOGIC: Scavenges ALL 3-digit numbered archives in vicinity
+                # This catches orphaned archives that could not be loaded into the ACV buffer due to overflow
+                for archive in archive_location.glob("*.[0-9][0-9][0-9]"):       # Scavenge the location of the archive to locate any tagged archive pending for compacting
+                    cls._compact_archive(archive)                                # Compact the archive into smaller size packet and disintegrate the archive
+                
+                cls._ACV_buffer_queue.task_done()                                # Indicate compact operation is completed , cross out one task in the todo list
+                
+            except queue.Empty:                                                  # If the ACV buffer is empty, move on
+                continue                                                         # Continue the cycle without doing anything
+            except Exception as e:                                               # If any other issue detected, identify with warning message
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)